README.md

```diff
@@ -55,6 +55,19 @@ provider_installation {
 EOF
 ```
 
+or if you are building locally (for development mode):
```

Comment:
this is redundant 
see the same block above

Reply:
removed